idflakies-core/src/main/java/edu/illinois/cs/dt/tools/detection/detectors/DetectorFactory.java

```diff
@@ -28,6 +29,8 @@ public static Detector makeDetector(final InstrumentingSmartRunner runner, final
             return new TuscanOnlyClassDetector(runner, baseDir, rounds, detectorType(), tests);
         } else if (detectorType().startsWith("alphabetical")) {
             return new AlphabeticalDetector(runner, baseDir, rounds, detectorType(), tests);
+        } else if (detectorType().equals("tuscan-intra-class")) {
```

Comment:
Maybe move this above to put next to tuscan (or move tuscan below to be next to this one).